src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,78 @@
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import classNames from 'classnames';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => (
+          <tr
```

Comment:
Here you could create separate component and map it, filling with necessary data. I'm not sure if it it's a good idea. But to be honest in my solution I did it becouse instruction in Readme file which says: "create the `<PersonLink person={person} />` and use it for all existing people") So what I'm trying to say is just give it another look;)